app/presenters/hyrax/presenter_factory.rb

```diff
@@ -56,11 +56,8 @@ def load_docs
 
       # Query solr using POST so that the query doesn't get too large for a URI
       def query(query, args = {})
-        args[:q] = query
-        args[:qt] = 'standard'
-        conn = Hyrax::SolrService.instance.conn
-        result = conn.post('select', data: args)
-        result.fetch('response').fetch('docs')
+        args[:method] = :post
```

Comment:
Can you comment about what's motivating these changes? Is this just a refactor, or would the straight replacement as below be broken?

```ruby
        args[:q] = query
        args[:qt] = 'standard'
        Hyrax::SolrService.post('select', data: args)
        result.fetch('response').fetch('docs')
```

Reply:
@no-reply I think the motivation is to replace the SolrService direct call with `.instance` here [conn = Hyrax::SolrService.instance.conn](https://github.com/samvera/hyrax/pull/3903/files/a123dfd3973d04a281e6a4d302a3611cf7c540ba#diff-c6712caea9e0ee2f6eff4a80283e48f1L61). This is just a refactor to use the method `Hyrax::SolrService.post(query, use_valkyrie: Hyrax.config.query_index_from_valkyrie, **args)` through `Hyrax::SolrService.query(query, args)` call.